kitsune/questions/utils.py

```diff
@@ -1,9 +1,12 @@
 import logging
 import re
 
+from django.conf import settings
+
 from kitsune.questions.models import Answer, Question
 
 log = logging.getLogger('k.questions')
+block_regex = re.compile(settings.QUESTION_BLOCK_REGEX) if settings.QUESTION_BLOCK_REGEX else None
```

Comment:
Shouldn't this be evaluated to False if settings is an empty string? Otherwise we return None instead of False in the `in_blocklist`. It will work but probably False is better.

Reply:
Ah yes. To fix this I changed the in_blocklist function to be a bit more explicit and return True/False, instead of Match/None